manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackActionTransformer.java

```diff
@@ -37,6 +43,8 @@ public BaseAction transform(BaseAction action, String bridgeId, String customerI
     // once we use a single topic for every connector there will be no need of having a different
     // one per connector https://issues.redhat.com/browse/MGDSTRM-5977
     private String generateKafkaTopicName(String processorId) {
-        return topicName;
+        return rhoasEnabled
+                ? String.format("ob-%s", processorId)
```

Comment:
I think that we can delete the old logic of the fixed topic name

```java
@ConfigProperty(name = "managed-connectors.topic-name")
String topicName;
```

now

Reply:
> 

After discussing with Alessandro we'll remove the fixed topic and the `rhoasEnabled` flag altogether